Source/Core/UICommon/UICommon.cpp

```diff
@@ -62,10 +62,25 @@ static void CreateDumpPath(const std::string& path)
   File::CreateFullPath(File::GetUserPath(D_DUMPTEXTURES_IDX));
 }
 
+static void CreateLoadPath(const std::string& path)
+{
+  if (!path.empty())
+    File::SetUserPath(D_LOAD_IDX, path + '/');
+  File::CreateFullPath(File::GetUserPath(D_HIRESTEXTURES_IDX));
```

Comment:
Why is this using 'Load/Textures/`?

Reply:
This matches the way "DumpPath" does it.  Any subdirectories underneath get created when the root path changes

Reply:
Ah yeah I see it a few lines above. Makes sense.